grumpy-rainbows/BangazonAPI
# Bm order controller

# Description

Creating controllers(GET, POST, PUT, and DELETE) for Order class including query parameters of include and completed. Also creating tests for each of the controllers as well as tests using each of the query parameters"

Fixes Issue # [Issue #5](https://github.com/grumpy-rainbows/BangazonAPI/issues/5)

## Type of change

Please delete options that are not relevant.

# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-order-controller`
3. Run all tests in TestOrder and verify that all tests work. They should work based on information already in the database we each have.


# Checklist:
